storage/src/rocksdb_storage/storage.rs

```diff
@@ -82,6 +82,34 @@ impl RocksDbStorage {
         res = blake3::hash(&res).as_bytes().to_vec();
         res
     }
+
+    // /// A helper method to build a prefix to rocksdb keys or identify a subtree
```

Comment:
I think this could be removed

Reply:
Hmmmm, best to keep it for now, just in case.